frontend/src/metabase/search/components/DropdownSidebarFilter/DropdownSidebarFilter.tsx

```diff
@@ -7,7 +7,7 @@ import type {
   SearchFilterDropdown,
   SearchFilterPropTypes,
 } from "metabase/search/types";
-import { Group, Text, Box, Button } from "metabase/ui";
+import { Group, Text, Box, Center, Button } from "metabase/ui";
```

Comment:
Found one more edge case that could happen because of specific font + language combination - the arrow down icon is not visible (label needs ellipsis or word-wrap)

![image](https://github.com/metabase/metabase/assets/6830683/bdea9f5d-60b3-4033-bd78-f986e87adabc)

Reply:
I'm going to make a few users in Thai/Polish/Hungarian/Ukrainian/etc to test these types of things in the future 😄